javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,197 @@
+  showFullName.classList.add("show");
+  return showFullName;
+}
+
+/*
+
+function fullName(firstName,surName){
+    return `${firstName} ${surName}`;
+}
+ */
+
+// Formal FullName
+
+const firstFormalName = document.getElementById("firstFormalName");
+const surFormalName = document.getElementById("surFormalName");
+const fullFormalNameButton = document.getElementById("fullFormalNameButton");
+const showFullFormalName = document.getElementById("showFullFormalName");
+const useFormalName = document.getElementById("useFormalName");
+
+function fullFormalName(firstFormalName, surFormalName, useFormalName) {
+  const fullName = useFormalName.checked
+    ? `Lord ${firstFormalName.value} ${surFormalName.value}`
+    : `${firstFormalName.value} ${surFormalName.value}`;
+  showFullFormalName.innerHTML = fullName;
+  showFullFormalName.classList.add("show");
+  return fullName;
+}
+
+// Event Application
+
+const currentDate = new Date();
+console.log(currentDate);
+console.log(currentDate.getDay());
+const todayNumber = currentDate.getDay();
+console.log(todayNumber);
+
+const daysToEvent = document.getElementById("daysToEvent");
+const calcDaysToEventButton = document.getElementById("calcDaysToEventButton");
+const showDayOfEvent = document.getElementById("showDayOfEvent");
+const daysOfWeek = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wedesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+function calcDaysToEvent() {
+  const days = parseInt(todayNumber) + parseInt(daysToEvent.value);
+  console.log(days);
+  const daysOfEvent = daysOfWeek[days % 7];
+  console.log(daysOfEvent);
+
+  showDayOfEvent.innerHTML = daysOfEvent;
+  showDayOfEvent.classList.add("show");
+  return daysOfEvent;
+}
+
+// Weather Wear
+
+const temperature = document.getElementById("temperature");
+const showWhatToWear = document.getElementById("showWhatToWear");
+
+function whatToWear(temperature) {
+  let answer = "";
+  temperature = temperature.value;
+  console.log(temperature);
+  if (temperature > 40) {
+    answer = "It is too hot, take off your clothes and stay home.";
+  } else if (temperature > 30) {
+    answer = "It is hot, wear hat,T-shirt and short and stay away from the Sun";
+  } else if (temperature > 15) {
+    answer = "It is nice, cool clothes, go out and injoy";
+  } else if (temperature > 5) {
+    answer =
+      "It is litte bite cold, have sweeter, but it is still a nice weather";
+  } else if (temperature > -5) {
+    answer = "It is cold , You shoul have a warm clothes and jacket";
+  } else if (temperature <= 5) {
+    answer = "It is so cold, Be warm and stay home";
+  } else {
+    answer = "Please, you should enter the temperature";
+  }
+  showWhatToWear.innerHTML = answer;
+  showWhatToWear.classList.add("show");
+  return answer;
+}
+
+// Student Manager
+
+const studentName = document.getElementById("studentName");
+const showStudentNumber = document.getElementById("showStudentNumber");
+let class015Students = [];
```

Comment:
This can still be a const key word to the variable, as you do not re-assign anything to the variable.
ex: const class015students = [];

class015students = ['A'];
In such cases as const would not allow this, you need to use "let" keyword, however here you just push the student name to the array, so it can still be const.
reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Look for: const in objects and arrays